### DISCUSSION TITLE
文章发布时间有问题

### DISCUSSION THREAD

**Discussion by shmilyabyy** [Tokinx/Wing#87] [问答]
![image](https://user-images.githubusercontent.com/77541484/205293687-d0590d17-f6a3-478d-90d0-f0fd8efef685.png)
![image](https://user-images.githubusercontent.com/77541484/205293851-03006d01-9710-417f-9a0e-a784a7a95101.png)
点进去文章发布的时间就会变成当天的零点时间。我发布这个”测试“的时间是2022/12/2 20:33，在笔记界面看是正常的，也就是几分钟前，但点进去文章就会变成20小时前，也就是2022/12/2 00:00发布的。请问这个是怎么回事呢？wp后台文章和页面的时间发布是正确的。

**Comment by Tokinx** [comment]
我将这个问题创建了issue，明天我会看一下。

**Comment by shmilyabyy** [comment]
非常感谢🙏

Jeff ***@***.***>于2022年12月3日 周六01:09写道：

**Comment by Tokinx** [comment]
该问题未复现
